src/module_library/maintenance_respiration.h

```diff
@@ -0,0 +1,128 @@
+#ifndef MAINTENANCE_RESPIRATION_H
```

Comment:
Can you run `clang-format` on this file? Some of the formatting is not standard (mostly related to blank spaces)